monitoring/monitorlib/messagesigning/uft.py

```diff
@@ -0,0 +1,7 @@
+def get_x_utm_jws_header(cert_url):
```

Comment:
It seems like we know a USS will need to specify their `kid`; why not accept that as a parameter now?